cmd/checkpoint/main.go

```diff
@@ -311,6 +334,28 @@ func getLocalParentPods() (map[string]*v1.Pod, error) {
 	return podListToParentPods(&podList), nil
 }
 
+// getLocalRunningPods uses the /runningpods/ kubelet api to retrieve the local container runtime pod state
+func getLocalRunningPods() (map[string]*v1.Pod, error) {
+	// TODO(aaron): The kubelet api is currently secured by a self-signed cert. We should update this to actually verify at some point
```

Comment:
I know it's in jest, but just to note: static pods can't use any api-server provided assets (secrets, config maps, etc.), so there's nothing to checkpoint even if it wanted to (also service accounts are secrets).

In the long term we have a few options. But ultimately what we need is a way to determine "are pods running" purely from local state:

- Create a `/runningpods/` like api on the insecure/read-only kubelet api
- Query the CRI interface (once used/available) to determine pod state directly from runtime
- Query runtime(s) directly